Gems/LyShine/Code/Include/LyShine/Animation/IUiAnimation.h

```diff
@@ -287,7 +287,7 @@ class UiAnimParamData
 //
 // Note: TCB splines are only for backward compatibility, Bezier is the default
 //
-enum EUiAnimCurveType
+enum EUiAnimCurveType : unsigned int
```

Comment:
Visual Studio 2022  converts the underlying type of this enum to `unsigned int` since it contains an entry with the value of `0xFFFFFFFF`
The change is to explicitly add the underlying type to the enum and then update the variables that compare against it to `unsigned int`